pkg/util/validation/limits.go

```diff
@@ -153,6 +153,7 @@ type Limits struct {
 	CompactorTenantShardSize           int            `yaml:"compactor_tenant_shard_size" json:"compactor_tenant_shard_size"`
 	CompactorPartialBlockDeletionDelay model.Duration `yaml:"compactor_partial_block_deletion_delay" json:"compactor_partial_block_deletion_delay"`
 	CompactorBlockUploadEnabled        bool           `yaml:"compactor_block_upload_enabled" json:"compactor_block_upload_enabled"`
+	CompactorVerifyChunks              bool           `yaml:"compactor_verify_chunks" json:"compactor_verify_chunks"`
```

Comment:
`CompactorBlockUploadVerifyChunks`. This flag should only control chunk verification during backfill.

(We should have similar option for enabling/disabling block verification here, but let's do that in a separate PR)